### DISCUSSION TITLE
Can not install in Appstore

### DISCUSSION THREAD

**Discussion by juliushuck** [iterate-ch/cyberduck#16573] [Q&A]
**Describe the bug**
I bought the app in the app store but cannot install it. It goes up to 60 % progress, then errors.

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS: macOS
 - Version: latest AppStore version

**Log Files**

**Additional context**

**Comment by juliushuck** [comment]
How do i get additional log files on the install from AppStore?

**Comment by tatze96** [comment]
Please check if you have another version of Cyberduck installed. To do so, open your _Applications_ folder and look for _Cyberduck.app_ files. Remove the file if one is present, and try installing Cyberduck from the App Store again.

If you don't have another version of Cyberduck installed, try restarting your operating system and restarting the installation process afterward.

If this doesn't work either, reach out to the App Store support for further troubleshooting in regard to the failed download. The licensing and application download is entirely handled by the App Store if you purchased Cyberduck from the App Store.

**Sidenote**: Replace Cyberduck with Mountain Duck within the aforementioned instructions if you are using Mountain Duck instead of Cyberduck.
